package.json

```diff
@@ -48,6 +48,7 @@
     "aes-js": "^3.1.2",
     "bootstrap": "^4.6.0",
     "bootstrap.native": "^3.0.15",
+    "gibberish-aes": "^1.0.0",
```

Comment:
We already have `aes-js` for doing AES things, no need to add a second library...

> Also, removing `gibberish-aes` as dependency was intentional (ref: #15).

Reply:
I added that to test something and forgot to remove that🤦🏾‍♂️